### ISSUE TITLE
The browsers examples only show how to create a db

### ISSUE THREAD

**Issue by shideneyu** [orbitdb/orbitdb-examples#2]
The browsers examples only show how to create a db

Unlike the nodes examples who show a real usecase (discovering nodes + replicating a database)

If I understand the most difficult thing on the browser environment is to have a relay node ready as to make the peer discovery possible to bootstrap the first nodes, would be nice to have it.

Thanks for this repository also, it is helpful !

**Comment by haydenyoung** [comment]
Yes, your assumption is correct. You will need a relay node to aid with address discovery on the browser node, basically saying, "This is the address which other nodes can connect to".

We are in the process of providing more extensive examples but there are currently time and resource constraints. In the meantime, I recommend looking at the unit tests for examples of how to connect browser nodes. In particular, check out the relay and dialling code under [test/utils](https://github.com/orbitdb/orbitdb/tree/main/test/utils).

**Comment by shideneyu** [comment]
I see. This link surely helps indeed. It's even more important to have exemple since many libp2p libraries are not available in a browser context (mdns...), making the whole thing difficult to test and setup, unless having to manually scavenge for hours on the internet.

I think I'm almost up in my understanding. Sure, we will be waiting for the update of this cool repo, thanks

**Comment by haydenyoung** [comment]
I also encourage you to join the helia and libp2p communities because it will a) allow you to be in direct contact with the developers who are probably best positioned to help with p2p best practices and b) bring both the ipfs and orbitdb communities closer together, hopefully resulting in some "cross-pollination" of ideas.

We have [a blog post here](https://orbitdb.substack.com/p/improving-orbitdb-connections-with) talking about the best methods for reaching out to the IPFS developers.
